.github/workflows/php.yml

```
@@ -66,4 +74,11 @@ jobs:
       run: composer install --prefer-dist --no-progress --ansi
 
     - name: Lint PHP files
+      if: ${{ matrix.php == 8 }}
       run: vendor/bin/phplint -w --exclude .git --exclude vendor --ansi .
+
+    - name: Check PHP syntax errors
+      if: ${{ matrix.php != 8 }}
+      uses: overtrue/phplint@9.6.2
```

```suggestion
      uses: overtrue/phplint@e2ebdbac93fa6ffe45d724e350f77df34c1f8fb4 #9.6.2
```

You should avoid tags and use commit ids.  Tags can be recreated with malicious code injected
The hashtag is just a comment, so we know what version we intended it to be agaisnt